### ISSUE TITLE
Add subject filename to classifications export

### ISSUE THREAD

**Issue by mkosmala** [zooniverse/Panoptes#940]
Currently, there's no way to associate a classification (by subject ID number in the export) with the subject I uploaded. Please add the file name of the subject to its metadata so that I can do associations. Otherwise, you'll have to require that science teams add a unique identifier in the metadata for each subject.

**Comment by camallen** [comment]
The plan is to provide a list of subject as well, see #886.

**Comment by mkosmala** [comment]
That's fine (see my comment in zooniverse/Panoptes-Front-End/issues/398), but make sure the subject filename appears as a metadata field in the subjects file.

**Comment by camallen** [comment]
@mkosmala just looked at a classification dump and the subject image_name and id is included in the json subject_data column. Is this not enough information to link the correct subject to the classification?

**Comment by mkosmala** [comment]
@camallen Has image_name perhaps been added since my dump on 5/28? I don't have it. I can send you my data dumps if you like.

**Comment by mkosmala** [comment]
Nope. Not in there yet. I now see 'retired' and 'workflow_name' (yay!), but still no image_name.

**Comment by camallen** [comment]
Ok - all subject metadata from the manifest upload **is** included in the export under *subject_data* column. It appears you didn't include the original filename as col in you manifest or the manifest uploader behaviour changed. Do you have your original manifest that you uploaded?

Looking at some data in the DB (demo site) metadata looks like `metadata: {"Camera"=>"XXXXXX", "Latitute"=>"", "Location"=>"", "Elevation"=>"", "Longitude"=>"", "Vegetation"=>""}`

**Comment by mkosmala** [comment]
R-i-g-h-t... I didn't do a manifest, per se. I manually created subjects with my script on the back-end -- because it was a long time ago. And, yes, I don't think I included filename in the metadata I sent the subject. So my bad. 

If I remember right, to create a subject, I first uploaded a file to S3, then got back a URL reference that I then passed to the Panoptes API. So at best, I could get that S3 address back out from each subject, right? (But not the original filename). I'm just thinking of how to match up my images with the results from the beta test...

Oh and also, here's a problem: our file names include timestamps, and I specifically **don't** want users to know the times the images were taken. Is there a way to hide certain metadata from being displayed by the metadata button? (Do I need to make a new issue?)
